.travis.yml

```diff
@@ -14,27 +14,22 @@ env:
     - PATH="${PYENV_ROOT}/shims:${PATH}"
 
 test_1_14: &test_1_14 >
-  ./build-support/ci.py
+  PEX_VERBOSE=9 ./build-support/ci.py
   --pants-versions 1.14.0
   --python-versions unspecified
 
 test_1_15: &test_1_15 >
-  ./build-support/ci.py
+  PEX_VERBOSE=9 ./build-support/ci.py
   --pants-versions 1.15.0
   --python-versions unspecified 2.7 3.6
 
 test_1_16: &test_1_16 >
-  ./build-support/ci.py
+  PEX_VERBOSE=9 ./build-support/ci.py
   --pants-versions config
   --python-versions unspecified 2.7 3.6 3.7
 
-test_1_16_without_py37: &test_1_16_without_py37 >
-  ./build-support/ci.py
-  --pants-versions config
-  --python-versions unspecified 2.7 3.6
-
 test_first_time_install: &test_first_time_install >
-  ./build-support/test_first_time_install.py
+  PEX_VERBOSE=9 ./pants test tests:first_time_install
 
 script:
   - *test_1_14
```

Comment:
The `ci.py` files should pass perfectly fine. I recommend not running those scripts to save CI time (about 6 minutes).

The only thing that fails is `*test_first_time_install`, because it's the only thing to trigger the `select-interpreter` task thanks to running `./pants test` with constraints defined in `pants.ini`.

Reply:
Yup, figured that out a bit too late. Thanks.
Nonetheless, the key data is now secured.